content/projects/gsoc/2019/project-ideas/artifact-promotion-plugin-for-jenkins-pipeline.adoc

```diff
@@ -13,7 +13,7 @@ skills:
 mentors:
 - "bicschneider"
 - "oleg_nenashev"
-- "meekrosoft"
+- "jonbrohauge"
```

Comment:
Add @MadsNielsen also?

Reply:
Not sure he wanted it. I'll ask if he wants on the list, and do a new PR.